Sources/Amplitude/Sessions.swift

```diff
@@ -112,10 +112,10 @@ public class Sessions {
 
     public func startNewSession(timestamp: Int64) -> [BaseEvent] {
         var sessionEvents: [BaseEvent] = Array()
-        let trackingSessionEvents = amplitude.configuration.trackingSessionEvents
+        let trackingSessionEvents = amplitude.configuration.defaultTracking.sessions
```

Comment:
Are we going to pump up a major version? otherwise should we support both?

Reply:
The SDK is in `Beta` status - breaking changes are possible. `trackingSessionEvents` is `true` by default - I think, most users don't set it explicitly.

Reply:
@yuhao900914 we plan to bump to 1.0 after DET and ObjC support are complete. Since this is still a beta version breaking changes are still possible. See https://www.docs.developers.amplitude.com/data/sdks/sdk-maintenance-and-support/#sdk-major-version-life-cycle